### ISSUE TITLE
GH Actions error

### ISSUE THREAD

**Issue by GretaTimaite** [udsleeds/openinfra#50]
It looks like there was a workflow run failure (commit [b712e8e](https://github.com/udsleeds/openinfra/commit/b712e8ebe6d4f31087948fedf637b2867179a71a)) when  @hulsiejames uploaded his Python scripts. Not 100% sure what caused it but would be good to examine it before it gets more complicated than needed (my personal experience).

What do you think @Robinlovelace?

**Comment by hulsiejames** [comment]
Looking up the warning from the failed push

```
── R CMD check results ─────────────────────────────── openinfra 0.0.0.9000 ────
Duration: 2m 22.9s

❯ checking for portable file names ... WARNING
  Found the following file with a non-portable file name:
    Python_scripts/Getting and using OSM data in Python.ipynb
  These are not fully portable file names.
  See section ‘Package structure’ in the ‘Writing R Extensions’ manual.

❯ checking installed package size ... NOTE
    installed size is 22.9Mb
    sub-directories of 1Mb or more:
      doc  22.8Mb

❯ checking top-level files ... NOTE
  Non-standard files/directories found at top level:
    ‘Python_scripts’ ‘lida_report’ ‘osm_highway_map.png’ ‘references.bib’
    ‘setup.R’
```
Specifically 
```
These are not fully portable file names.
See section ‘Package structure’ in the ‘Writing R Extensions’ manual.
```
From section [Package Structure](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Package-structure) in Writing R Extensions

_"we exclude space as many utilities do not accept spaces in file paths"_

It seems a rookie mistake on my part :upside_down_face:

I will change the filename / delete the existing file and re-upload with spaces removed from the filename and hope that works

**Comment by GretaTimaite** [comment]
hahaha, happens all the time. It's great it's a quick fix.

**Comment by GretaTimaite** [comment]
Thanks, @hulsiejames for sorting this out so quickly.

**Comment by Robinlovelace** [comment]
Good fix guys, sorry for picking up on it a bit late but great to see good workflow in reporting/diagnosing/fixing issues.

Final tip on that path, if you create commit messages with the word 'fix' you can fix the issue, e.g.

```
git add .
git commit -am 'Fix #50'
git push
```

Would fix the issue: https://github.blog/2013-01-22-closing-issues-via-commit-messages/
